src/types/Offer.js

```diff
@@ -13,17 +13,21 @@ import BaseValidator from './base.js';
 
 export default class PriceSpecificationValidator extends BaseValidator {
   getConditions() {
-    return [
-      this.or(
-        this.required('price', 'number'),
-        this.required('priceSpecification.price', 'number'),
-      ),
-      this.recommended('availability'),
-      this.or(
-        this.recommended('priceCurrency', 'currency'),
-        this.recommended('priceSpecification.priceCurrency', 'currency'),
-      ),
-      this.recommended('priceValidUntil', 'date'),
-    ].map((c) => c.bind(this));
+    const conditions = [];
+    if (this.path.length === 2 && this.path[0].type === 'Product') {
```

Comment:
This conditions limits the rules to `Product -> Offer`. There's also the case in `ProductGroup` where it's `ProductGroup -> Product -> Offer`.

See https://developers.google.com/search/docs/appearance/structured-data/product-variants.

Please adjust the condition so that it only requires the parent of `Offer` to be a `Product`.